docs/osf/setup.rst

```diff
@@ -18,21 +18,29 @@ Before you can begin to install the OSF, you will first need to install several
 Installing Homebrew
 -------------------
 
-Homebrew is a package manager that allows you to install lots of very cool things that are not just python related. You most likely have homebrew.
-
 Homebrew is a package manager that allows you to install many cool things easily (not just python tools)- it will greatly ease the process of installing OSF requirements. To see if Homebrew is already installed, open a new window in your terminal and type
 
     ::
 
         brew
 
-If you see a list of options you already have homebrew and you can skip this section. If not you will want to install homebrew globally, not just in your osf environment. To install it, open a new terminal window and run the following command.
+If you see a list of options you already have homebrew.  Homebrew prefers to be run by one user, so you'll need to take ownership of it and homebrew-cask.
+
+    ::
+
+        sudo chown -R <your username> /usr/local
```

Comment:
Most devs using their own machine will not--and *should* not--need to do this. Perhaps this should be moved to the Common Errors section.